Capstone/News/NewsViewController.m

```diff
@@ -8,33 +8,63 @@
 #import "CollegeNews.h"
 #import "NewsCell.h"
 #import "APIManager.h"
+#import "ParseCollege.h"
+
 @interface NewsViewController ()<UITableViewDelegate,UITableViewDataSource>
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *activityIndicator;
+@property (strong, nonatomic) UIRefreshControl *refreshControl;
 @end
 @implementation NewsViewController {
     NSMutableArray<CollegeNews *> *collegeArticles;
+    NSString *websitesToGetNewsFrom;
 }
 
 - (void)viewDidLoad {
     [super viewDidLoad];
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
     self->collegeArticles = [[NSMutableArray alloc] init];
-    [self fetchData];
+    self->websitesToGetNewsFrom = [[NSString alloc] init];
+    self.refreshControl = [[UIRefreshControl alloc] init];
+    [self.refreshControl addTarget:self action:@selector(fetchUserLikes) forControlEvents:UIControlEventValueChanged];
+    [self.tableView insertSubview:self.refreshControl atIndex:0];
+    [self fetchUserLikes];
+    [self.activityIndicator startAnimating];
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [self fetchUserLikes];
 }
 
-- (void)fetchData {
-    [[APIManager shared] fetchCollegeNews:^(NSArray *collegeNews, NSError *error) {
-        if (error != nil) {
-            NSLog(@"%@", error);
+- (void)fetchUserLikes {
+    PFUser *currentUser = [PFUser currentUser];
+    PFRelation *relation = [currentUser relationForKey:@"likes"];
```

Comment:
Let's clarify the naming so that what `relation` is is clearer later one in this method
```suggestion
    PFRelation *likesRelation = [currentUser relationForKey:@"likes"];
```